checks/vulnerabilities.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2021 Security Scorecard Authors
+// Copyright 2022 Security Scorecard Authors
```

Comment:
```suggestion
// Copyright 2021 Security Scorecard Authors
```

No need to change existing copyright dates.

Reply:
sure, what's the reasoning behind this? I've basically replaced the entire file with new code. I'm fine not updating, I just want some clarification so that I can make an informed decision in future PRs. Thanks

Reply:
To be fair, my reasoning was roughly, "this is what we do in Kubernetes", which isn't necessarily legal advice.
For Kubernetes, we only note the year in which the file was first created.

Here's some guidance from ASF: https://www.apache.org/legal/src-headers.html

Also tagging @david-a-wheeler @swinslow for opinions.

Reply:
(@justaugustus for visibility, I'm no longer employed by the LF, so I'm not speaking on their behalf here in any official way)  :)

The LF's general guidance to projects over the past few years has been to recommend a copyright notice form that omits the year. I had previously written up [a blog post](https://www.linuxfoundation.org/blog/copyright-notices-in-open-source-software-projects/) that describes a "generalized" copyright notice format in some detail, which was related to [earlier guidance for CNCF](https://github.com/cncf/foundation/blob/main/copyright-notices.md).

The main reason for this recommendation was to avoid the sort of question raised here: e.g. developers wondering every January whether or not to bump the notice up to the next year; with the real answer depending on an analysis of copyrightability of particular contributions that generally goes beyond what folks want to get into. And since the year is not mandatory in order for a work to be copyrighted, some projects have made the decision to omit it.

Keep in mind the caveats that (1) there's nothing _wrong_ with including the year, this was just one recommendation; and (2) you'd never want to modify a copyright notice that names a particular third party without their consent.

Hope this helps  :)

Reply:
Thanks for the feedback. Let's discuss this in an issue and address it via a PR if we decide to update the dates or remove them entirely. @justaugustus can you create a tracking issue?

Reply:
> (@justaugustus for visibility, I'm no longer employed by the LF, so I'm not speaking on their behalf here in any official way) :)

@swinslow --I had completely forgotten, but thanks so much for the providing some guidance nonetheless. Hope you're well! :)

> Thanks for the feedback. Let's discuss this in an issue and address it via a PR if we decide to update the dates or remove them entirely. @justaugustus can you create a tracking issue?

Absolutely. Opened https://github.com/ossf/scorecard/issues/1534 for this.